heudiconv/cli/run.py

```diff
@@ -248,11 +248,11 @@ def process_args(args):
 
     outdir = op.abspath(args.outdir)
 
-    import etelemetry
     try:
+        import etelemetry
         latest = etelemetry.get_project("nipy/heudiconv")
     except Exception as e:
-        lgr.warning("Could not check for version updates: ", e)
+        lgr.warning("Could not check for version updates: %s", str(e))
```

Comment:
```suggestion
        lgr.warning("Could not check for version updates: %s", str(e))
        lgr.warning(("Could not check for version updates: ", e)) 
```

Reply:
this would log it as a tuple.  Also docstring says:

```
In [4]: log.warning?
Signature: log.warning(msg, *args, **kwargs)
Docstring:
Log 'msg % args' with severity 'WARNING'.

To pass exception information, use the keyword argument exc_info with
a true value, e.g.

logger.warning("Houston, we have a %s", "bit of a problem", exc_info=1)
```
so it expects a string as the first argument, not a tuple.  It actually works if I pass a tuple, but e.g. that would have caused datalad, which provides custom format which expects str, to puke.  So I would keep it as it was